usaspending_api/etl/management/commands/load_subawards.py

```diff
@@ -60,17 +62,29 @@ def handle(self, *args, **options):
         else:
             db_cursor = PhonyCursor()
 
-        submission_ids_to_update = []
+        submissions_to_update = []
 
         if options["update_all"]:
             submissions_to_update = SubmissionAttributes.objects.exclude(broker_submission_id__isnull=True)
         else:
-            submission = SubmissionAttributes.objects.filter(broker_submission_id=options['submission_id']).first()
-            if not submission:
-                logger.critical("Submission {} not found in datastore".format(options['submission_id']))
-                return
-            submissions_to_update = [submission]
+            for submission_id in options['submission_id']:
+                sub = SubmissionAttributes.objects.filter(broker_submission_id=submission_id).first()
```

Comment:
Gotcha--that makes sense. I might go rogue and sneak in a PR for that unique index.